utils/notify/publish_notification.py

```diff
@@ -0,0 +1,53 @@
+import os
+
+from slack_sdk import WebClient
+from slack_sdk.models import blocks
+
+
+class Settings:
+    channel: str
+    workspace: str
+
+
+class PublishNotify:
+    settings: Settings
+
+    def __init__(self, send_notifications: bool = True):
+        self.settings = Settings()
+        self.send_notifications = send_notifications
+        self.release_notes_url = "https://github.com/uktrade/platform-tools/releases/latest"
```

Comment:
Should this be a constant at the top of the file?

Reply:
yes, fair point; separated it just because it's a long url, but we can move into a const